independent-publisher-connectors/OpenAI/apiDefinition.swagger.json

```diff
@@ -87,20 +87,25 @@
             "name": "engine",
             "in": "path",
             "type": "string",
-            "description": "The used engine, choose between text-davinci-002, text-curie-001, text-babbage-001, text-ada-001",
+            "description": "The used engine, choose between text-davinci-003, text-davinci-002, text-curie-001, text-babbage-001, text-ada-001",
             "required": true,
-            "default": "text-davinci-002",
+            "default": "text-davinci-003",
             "x-ms-visibility": "important",
             "x-ms-summary": "Engine",
             "enum": [
+              "text-davinci-003",
               "text-davinci-002",
               "text-curie-001",
               "text-babbage-001",
               "text-ada-001"
             ],
             "x-ms-enum-values": [
               {
-                "displayName": "DaVinci - most expensive",
+                "displayName": "DaVinci 003 - added in 12/2022, better than 002 at same cost",
+                "value": "text-davinci-002"
```

Comment:
@PowerRobin The 'x-ms-enum-values' extension has duplicate values. You can only have one display name mapped to a value. Line 105 and line 109.

Reply:
Oh yes, line 105 has to be "text-davinci-003", my bad.
Fixed it!